app/controllers/health_monitor/health_controller.rb

```diff
@@ -25,7 +25,7 @@ def check
     private
 
     def statuses
-      res = HealthMonitor.check(request: request)
+      res = HealthMonitor.check(request: request, params: params)
```

Comment:
Can you make sure to sanitize params (e.g., using strong parameters) to only support `providers` for now and nothing else?